neo-starter-rabbitmq/src/main/java/com/neoteric/starter/rabbit/RabbitAdditionalAutoConfiguration.java

```diff
@@ -45,8 +45,6 @@
 @EnableConfigurationProperties(StarterRabbitProperties.class)
 public class RabbitAdditionalAutoConfiguration {
```

Comment:
Can you please change name to StarterRabbitAutoConfiguration (and spring.factories) ? Let's stick with this naming convention for now (in case where unable to find better name)